README.md

```diff
@@ -1,2 +1,15 @@
 # ion-blender-exporter
 Blender add-on for uploading and tiling models with Cesium ion. https://cesium.com
+
+## Installation
+1. Download the project as a zip [here](https://github.com/AnalyticalGraphicsInc/ion-blender-exporter/archive/beta.zip)
+2. Go to `File > Preferences > Add-ons`
+3. Enable Add-on
+
+## Todo
```

Comment:
We don't keep TODO lists in the README.  You can move these into the PR description for now (we normally use task lists for this kind of stuff).  For stuff not part of the PR, we should write up new issues.

Reply:
Yeah I thought it would help contextualize where I am taking the codebase

Reply:
That's a valid approach, just not how we use READMEs throughout Cesium. At long as it's cleaned up before we merge, that's fine.